client/src/app/employee/employee-detail/state/employee.service.ts

```diff
@@ -1,12 +1,14 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { ToastrService } from 'ngx-toastr';
 import { throwError } from 'rxjs';
 import { catchError } from 'rxjs/operators';
-import { EnvService } from 'src/app/shared/env.service';
+
 import { Employee, EmployeeUpdate } from './employee.model';
 import { EmployeeStore } from './employee.store';
 
+import { ToastrService } from 'ngx-toastr';
+import { EnvService } from 'src/app/shared/env.service';
```

Comment:
This import path seems wrong - Angular/Typescript should be doing the import from `'./app/shared/env.service'`. I wonder why it sees `src` as the root.

We can set up import aliases so that we'll have absolute imports from `'@payrolls/...'`, `'@shared/...'`

Reply:
It looks like this is happening in multiple places for EnvService. How do I set up `@shared` as an import alias?

Reply:
https://github.com/microsoft/TypeScript/issues/39666

Something is broken in VS Code or Typescript and it's related to these new "Solution Style" `tsconfig.json` files.

We can switch back to the previous `tsconfig.json` and then import aliases (and `import` intellisense) will work.

Reply:
Looks like the `tsconfig.json` setup that the Angular 10 CLI produces has issues currently and it doesn't appear they will resolved soon.

I switched the structure to use the older `tsconfig.json` setup and added the import path aliases.

Reply:
Nice! Issues with angular 10 CLI, not good!